examples/make_eeg_bids.py

```diff
@@ -0,0 +1,181 @@
+data_dir = _TempDir()
+
+###############################################################################
+# Now, we download data for participant sub-05 (about 350MB). We could also
+# download the whole dataset by specifying `url_dict` as an input to
+# `download_matchingpennies_subj`, see the function's docstring for more
+# information.
+download_matchingpennies_subj(directory=data_dir)
+
+###############################################################################
+# Let's see whether the data has been downloaded using a quick visualization
+# taken from https://stackoverflow.com/a/16974952/5201771
+for root, dirs, files in os.walk(data_dir):
+    path = root.split(os.sep)
+    print((len(path) - 1) * '-----', os.path.basename(root))
+    for file in files:
+        print(len(path) * '-----', file)
+
+###############################################################################
+# The data are part of the example suit for the EEG specification of BIDS, so
+# they are already in BIDS format as we can see from the file names (also, see
+# the `channels.tsv` and `events.tsv` files). However, we want to start from
+# scratch and will only use the "pure" EEG data in its BrainVision format with
+# a '.eeg' for binary data, a '.vhdr' for header information, and a '.vmrk' for
+# specifying the event markers within the data.
+
+###############################################################################
+# Step 2: Formatting as BIDS
+# --------------------------
+#
+# We are reading the data using MNE-Python's io module and the
+# `read_raw_brainvision` function, which takes a `.vhdr` file as an input.
+vhdr_path = os.path.join(data_dir, 'sub-05_task-matchingpennies_eeg.vhdr')
+raw = read_raw_brainvision(vhdr_path)
+print(raw)
+
+###############################################################################
+# By reading the raw data, MNE-Python has also read the `.vmrk` file in which
+# the events of the EEG recording are stored. It has automatically created a
+# new "stimulus channel" containing the event pulses and appended this to our
+# data:
+print(raw.ch_names)  # Should be called something like "STI"
+
+###############################################################################
+# For BIDS, we specify all events in a dedicated `events.tsv` file. Let's read
+# the events from our stimulus channel using `find_events`
+events = find_events(raw)
+
+###############################################################################
+# Now we have everything to start a new BIDS directory using our data.
+# To do that, we can use the high level function `raw_to_bids`, which is the
+# core of MNE-BIDS. Generally, `raw_to_bids` tries to extract as much meta data
+# as possible from the raw data and then format it in a BIDS compatible way.
+# `raw_to_bids` takes a bunch of inputs, most of which are however optional.
+# The required inputs are described in the docstring:
+#
+# .. code-block:: python
```

Comment:
why not use sphinx gallery separators with the # symbol

Reply:
I don't really know what you mean by that --> I just want to show the docstring of the required inputs. 

You mean I should do that using actual code, like `help(raw_to_bids)`? That would print out the entire docstring, whereas I only want to show the non-defaulting parameters that HAVE to be specified.

Reply:
oh I see, then I'd just remove this code block ...